azure-pipelines.yml

```diff
@@ -14,6 +14,17 @@ pr:
 - features/*
 - demos/*
 
+variables:
+    # Disable component governance in our CI builds. These builds are not shipping nor
+    # are they a service. Also the component governance jobs issue lots of inconsequential
+    # warnings and errors into our build timelines that make it hard to track down 
+    # real errors in the build
```

Comment:
Does this mean any legitimate component governance issue isn't going to get flagged until a signed build?

Reply:
These jobs don't actually issue errors though because we're not a service. The errors they do issue are when the service itself fails. If they did start issuing legitimate errors I'd consider turning them back on.

Reply:
I thought this was the thing that also tracked our NuGet dependencies for security tracking? Or is that something else?

Reply:
If it's not that's fine, I just wonder where *that* is then... :smile: